pkg/sink/sink.go

```diff
@@ -72,27 +72,39 @@ func (r Resource) HandleEvent(response http.ResponseWriter, request *http.Reques
 	log.Printf("EventListener: %s in Namespace: %s handling event (EventID: %s) with payload: %s and header: %v",
 		r.EventListenerName, r.EventListenerNamespace, eventID, string(event), request.Header)
 
+	result := make(chan int, 10)
 	// Execute each Trigger
 	for _, trigger := range el.Spec.Triggers {
-		go r.executeTrigger(event, request, trigger, eventID)
+		go r.executeTrigger(event, request, trigger, eventID, result)
+	}
+
+	code := http.StatusAccepted
+	for i := 0; i < len(el.Spec.Triggers); i++ {
+		thiscode := <-result
+		if thiscode < code {
+			code = thiscode
+		}
 	}
 
 	// TODO: Do we really need to return the entire body back???
+	response.WriteHeader(code)
```

Comment:
Please add a test in [`sink_test.go`](https://github.com/tektoncd/triggers/blob/master/pkg/sink/sink_test.go#L489) to check the `statusCode` of the response.

I think it'll just be a quick check of [this](https://github.com/tektoncd/triggers/blob/master/pkg/sink/sink_test.go#L489) `resp.statusCode`.

Reply:
I added the check.